### ISSUE TITLE
Display TensorFlow/PyTorch versions for NGC images

### ISSUE THREAD

**Issue by adrysn** [lablup/backend.ai-console#569]
NGC image's tag is composed of the release date of the image. So, by looking at the image tags, it is hard to know what version of ML libraries, such as TensorFlow, PyTorch, are baked into the image. It would be good to show the library versions as well.

NGC images already labeled those information, and those are stored on the Backend.AI's Etcd server. For example, a client can get the information by requesting to `/config/get` API with following locations as parameter.

* TensorFlow: `images/index.docker.io/lablup%2Fngc-tensorflow/19.10-py3/labels/com.nvidia.tensorflow.version`
* PyTorch: `images/index.docker.io/lablup%2Fngc-pytorch/19.10-py3/labels/com.nvidia.pytorch.version`

**Comment by inureyes** [comment]
Technical issues are resolved. However, UI issues exist. Where to show the information?

<img width="419" alt="image" src="https://user-images.githubusercontent.com/949915/84160001-abb1b800-aaa8-11ea-8910-51f738817541.png">

**Comment by inureyes** [comment]
Now the framework information overrides base version.
